root/edit/details/historic/RemovePuid.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import DescriptiveLink
+  from '../../../static/scripts/common/components/DescriptiveLink';
+
+type RemovePuidEditT = {
+  ...EditT,
+  +display_data: {
+    puid_name: string,
+    recording: RecordingT,
+  },
+};
+
+const RemovePuid = ({edit}: {edit: RemovePuidEditT}) => (
+  <>
```

Comment:
The surrounding `<></>` can be removed since the component only returns one element